pilot/pkg/features/experimental.go

```diff
@@ -29,7 +29,8 @@ var (
 	FilterGatewayClusterConfig = env.Register("PILOT_FILTER_GATEWAY_CLUSTER_CONFIG", false,
 		"If enabled, Pilot will send only clusters that referenced in gateway virtual services attached to gateway").Get()
 
-	SendUnhealthyEndpoints = atomic.NewBool(env.Register(
+	// RawSendUnhealthyEndpoints contains the raw setting on RawSendUnhealthyEndpoints. This should be checked per-service
+	RawSendUnhealthyEndpoints = atomic.NewBool(env.Register(
```

Comment:
nit: GlobalSendUnhealthyEndpoints or some thing similar - RawSendUnhealthyEndpoints not conveying proper meaning